providers/cncf/kubernetes/docs/changelog.rst

```diff
@@ -28,6 +28,35 @@ Changelog
 ---------
 
 
+10.9.0
+......
+
+Features
+~~~~~~~~
+
+* ``Move container-related functions from PodManager to a separate file (#56700)``
```

Comment:
This may be unintentionally breaking change(?) I left a comment on the PR to verify

would appreciate someone else to take a look